circle.yml

```diff
@@ -15,15 +16,16 @@ dependencies:
     - "~/.cabal"
 
   override:
-    - mkdir -p /home/ubuntu/.go_workspace/src/github.com/Originate
-    - cp -r /home/ubuntu/git-town /home/ubuntu/.go_workspace/src/github.com/Originate/gt
-    - cd /home/ubuntu/.go_workspace/src/github.com/Originate/gt && go install
-    - rm -rf /home/ubuntu/.go_workspace/src/github.com/Originate/gt
+    - mkdir -p "$(dirname "$GO_PROJECT")"
+    - if [[ ! -e "$GO_PROJECT" ]]; then ln -s $HOME/git-town "$GO_PROJECT"; fi
+    - cd $GO_PROJECT && go install && go get gopkg.in/alecthomas/gometalinter.v1
+    - gometalinter.v1 --install
     - cd /home/ubuntu/git-town && bundle install
```

Comment:
No, on Circle the source code is in `./git-town`, so we have to cd into it. When inferring the command, Circle does that, but here we run it manually, so have to change the directory.